spec/Conversions.tex

```diff
@@ -236,6 +248,24 @@ \subsection{Implicit Class Conversions}
 Third, an implicit conversion from class type \chpl{D} to another class
 type \chpl{C} is allowed when \chpl{D} is a subclass of \chpl{C}.
 
+Any combination of these three conversions is allowed.
+
+\subsection{Implicit Type Argument Conversions}
+\label{Implicit_Type_Arg_Conversions}
+\index{conversions!implicit!type arguments}
+
+When the actual is a type, it is passed to a formal of a \chpl{type} intent,
```

Comment:
"When the actual is a type, it is passed to a formal of a \chpl{type} intent, including the \chpl{this} formal of a type method."

feels like it is missing something. How about:

An implicit type argument conversion applies when a type actual is passed to a formal with type intent. This includes the this formal of a type method.